packages/upset/src/components/AltTextSidebar.tsx

```diff
@@ -147,111 +114,136 @@ export const AltTextSidebar: FC<Props> = ({ open, close, generateAltText }) => {
-                </Box>
-              </Box>
-              <Box>
-                <Box sx={plotInfoItem}>
-                  <Typography variant="h4" sx={plotInfoTitle}>
-                    Items:
-                  </Typography>
-                  <TextField
-                    onChange={(e) => setPlotInformation({ ...plotInformation, items: e.target.value })}
-                    sx={{ width: '70%' }}
-                    multiline
-                    InputLabelProps={{ shrink: true }}
-                    value={plotInformation.items}
-                    fullWidth
-                    maxRows={8}
-                    disabled={!isEditable}
-                    placeholder={`eg: ${placeholderText.items}`}
-                  />
-                </Box>
-              </Box>
-              <Typography variant="body1">{generatePlotInformationText()}</Typography>
-              { isEditable && <Button color="error" onClick={handleEditableChange}>Cancel</Button>}
-              { isEditable && <Button onClick={() => { actions.setPlotInformation(plotInformation); setIsEditable(false); }}>Save</Button> }
-            </AccordionDetails>
-          </Accordion>
-        </Box>
-        <Box marginTop={2}>
-          <div css={css`overflow-y: auto; padding-bottom: 4rem;`}>
-            {useLong && !textGenErr && <Button onClick={() => setUseLong(false)}>Show Less</Button>}
-            <ReactMarkdownWrapper text={altText ? useLong ? altText.longDescription : altText.shortDescription : ''} />
-            {!useLong && !textGenErr && <Button onClick={() => setUseLong(true)}>Show More</Button>}
-          </div>
+        <br />
+        <Typography variant="h1" fontSize="1.4em" fontWeight="inherit" height="1.4em" padding="0">
+          Accessibility Sidebar
+        </Typography>
+        {divider}
+        <PlotInformation divider={divider} tabIndex={10} />
+        <Typography variant="h2" fontSize="1.2em" fontWeight="inherit" height="1.4em" padding="0" marginTop="1em">
+          Description
+        </Typography>
+        {divider}
+        <Box marginTop={2} css={css`overflow-y: auto;`}>
+          {!textGenErr ? (<>
+            <FormControlLabel
+              style={{marginLeft: "5px"}} // Align with below text; has 2px border & 3px padding
+              sx={{ '& span': { fontSize: '0.8rem' } }} // Fontsize can't be set in style prop for some reason
+              label="View User Description(s)"
+              control={
+                <Switch
+                  size="small"
+                  style={{marginRight: "10px"}}
+                  checked={currState.useUserAlt || textEditing}
+                  tabIndex={9}
+                  onChange={(ev) => {
+                    if (currState.useUserAlt !== ev.target.checked)
+                      actions.setUseUserAltText(ev.target.checked);
+                    if (!ev.target.checked) {
+                      setTextEditing(false);
+                    }
+                  }}
```

Comment:
Consider moving onclick functions with logic into handler functions. Not critical

Reply:
Generally I only do this if the functionality is duplicated or there's a ton of logic; I don't think that's the case here.